src/index.js

```diff
@@ -149,10 +172,15 @@ export default (options = {}) => {
 
         const concat = new Concat(true, fileName, '\n')
         const entries = [...extracted.values()]
-        const { modules } = bundle[normalizePath(path.relative(dir, file))]
+        const { modules, facadeModuleId } = bundle[
```

Comment:
`modules` here should already be in the deterministic import order, but in rollup v2 will exclude tree-shaken modules. 

If you're using rollup 2.21.0, we should be able to modify the return value of the transform hook, to ensure css code doesn't get shaken away and thus excluded from `modules`:

```diff
return {
        code: result.code,
        map: result.map || { mappings: '' },
        moduleSideEffects: 'no-treeshake',
}
```

I thiiiink that means we won't need to do this the sorting dance as the ordered list of css modules is something like the following for multiple bundles:

```
const moduleIds = flatMap(Object.values(bundle), (fileInfo) =>
  Object.keys(fileInfo.modules),
).filter(loaders.isSupported(id));
```

Reply:
That would be amazing! I'll try to see if I can get it to work.

Reply:
Seems like `modules` also don't work, they show files in random order for me in local rollup v2 tests

Reply:
@vladshcherbin could you define "unexpected order"? Could you provide an example of what your expected order would be?

Reply:
@BPScott Sure, I've added a comment in https://github.com/rollup/rollup/issues/3682#issuecomment-667307596 with the order I'd expect to have. It's the same as @wardpeet mentioned as expected in linked issue.

Reply:
I've enabled both methods now and added the no-treeshake for newer versions of rollup.